Riot/Modules/Common/Recents/DataSources/RecentsDataSource.m

```diff
@@ -823,57 +804,6 @@ - (UIView *)viewForHeaderInSection:(NSInteger)section withFrame:(CGRect)frame
     [sectionHeader addSubview:headerLabel];
     sectionHeader.headerLabel = headerLabel;
 
-    if (section == directorySection
-        && _recentsDataSourceMode == RecentsDataSourceModeRooms
-        && !(shrinkedSectionsBitMask & RECENTSDATASOURCE_SECTION_DIRECTORY)
-        && BuildSettings.publicRoomsAllowServerChange)
-    {
-        if (!directorySectionContainer)
-        {
-            directorySectionContainer = [[DirectorySectionHeaderContainerView alloc] initWithFrame:CGRectZero];
-            directorySectionContainer.backgroundColor = [UIColor clearColor];
-
-            // Add the "Network" label at the left
-            networkLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, 100, 30)];
-            networkLabel.font = [UIFont systemFontOfSize:16.0];
-            networkLabel.text = NSLocalizedStringFromTable(@"room_recents_directory_section_network", @"Vector", nil);
```

Comment:
This string can be removed from translations?

Reply:
Ah good spot.